services/docker-compose.devel.yml

```diff
@@ -73,7 +76,7 @@ services:
       - "3006:3000"
     environment:
       - SC_BOOT_MODE=debug-ptvsd
-      - SIDECAR_LOGLEVEL=DEBU
+      - SIDECAR_LOGLEVEL=DEBUG
```

Comment:
DEBUG mode for that one and not the standard sidecar is that correct?

Reply:
being used for development, it's ok to use it in debug (other services in this file have also debug enabled)